docs/modules/deployment/pages/upgrade/basic.adoc

```diff
@@ -0,0 +1,134 @@
+
+Your {page-component-title} repository is defined in a file in `/etc/yum.repos.d/`.
+It may be named {page-component-title}.repo  but is not guaranteed to be.
+
+. Use the text editor of your choice to edit your {page-component-title} repository config to change the `baseurl` of the repository from the old release family to the new release family.
+. Verify that the attribute `enabled=1` is defined, or enabled, or absent entirely
+. Purge any cached yum data: `yum clean all`
++
+Expected outcome: `yum repolist` does not show an error pertaining to the {page-component-title} repo; `yum info {page-component-title}-core` shows an available current package version.
+
+. Install the PostgreSQL 12 version of iplike: `yum -y install iplike-pgsql12`
++
+Expected outcome: `rpm -q iplike-pgsql12` reports an _iplike-pgsql12_ package is installed.
+
+. Start postgresql-12: `systemctl start postgresql-12.service`
++
+Expected outcome: `systemctl status postgresql-12.service` reports postgresql-12 has started.
+
+. Make a backup copy of your OpenNMS config: `mv /opt/opennms/etc /opt/opennms/etc.backup`
+
+. Upgrade the {page-component-title} packages to the new version: `yum -y upgrade {page-component-title}-core {page-component-title}-webapp-jetty`
++
+Expected result: A lot of rpmnew files; `rpm -q {page-component-title}-core` reports the new version is installed.
+
+. Move the package-updated etc directory out of the way: `mv $OPENNMS_HOME/etc $OPENNMS_HOME/etc.as.shipped.2021`
+
+. ##What do users without support tarball do?## Unpack the OpenNMS Support-provided config tarball in `/opt/opennms`: `cd /opt/opennms; tar zxvf apple_etc.tar.gz`
++
+Expected result: a new /etc directory is created.
+
+. Install Java 11: `yum -y install java-11-openjdk java-11-openjdk-devel`
++
+Expected result: `rpm -q java-11-openjdk` returns a result.
+
+. Execute runjava:
+`$OPENNMS_HOME/bin/runjava -S /usr/lib/jvm/java-11/bin/java`
++
+Expected outcome: `$OPENNMS_HOME/etc/java.conf` contains a path to the Java 11 JDK.
+
+. Run the opennms installer: `$OPENNMS_HOME/bin/install -dis`
++
+NOTE: This step took approximately two hours in a lab environment processing 1.9 million database events.
+It may be faster on your own hardware, but may still take significant time.
++
+Expected result: A long delay, followed by output containing: `Upgrade completed successfully!`
+
+. Clear the Karaf cache: `yes | $OPENNMS_HOME/bin/fix-karaf-setup.sh`
++
+Expected outcome: The directory `$OPENNMS_HOME/data/` is empty.
+
+. Start OpenNMS {page-component-title}: `systemctl start opennms.service`
++
+Expected result: OpenNMS {page-component-title} starts with no errors and the Web UI is accessible as normal.
++
+NOTE: `tail -F /var/log/opennms/manager.log` can illustrate at what point in the startup process {page-component-title} is currently.
+
+. Upgrade is complete and operation is resumed.
+
+##when do they copy over their changed config files?##
```

Comment:
If it's upgraded in place, they don't.  Any shipped config that conflicts with existing user-modified config gets renamed to <filename>.rpmnew, and these files have to be manually inspected and the differences reconciled.  I like to use `diff -Bbw` and `diff -y` if I'm doing it by hand.